### ISSUE TITLE
Can't start the app

### ISSUE THREAD

**Issue by eson57** [Rikujolla/draftmap#1]
Well, it will not start. I can see something is missing, but I do not really understand.
```
| SailfishOS 2.1.0.11 (Iijoki) (armv7hl)
'---
[nemo@Sailfish ~]$ harbour-kuvakartta
[D] unknown:0 - Using Wayland-EGL
[W] unknown:39 - file:///usr/share/harbour-kuvakartta/qml/harbour-kuvakartta.qml:39:30: Type MapPage unavailable 
         initialPage: Component { MapPage { } } 
                                  ^
[W] unknown:30 - file:///usr/share/harbour-kuvakartta/qml/pages/MapPage.qml:30:1: module "QtLocation" is not installed 
     import QtLocation 5.0 
     ^

```

**Comment by Rikujolla** [comment]
Thanks for the info. I will look at this. For me its ok, but I may have QtLocation installed in my phone due to Rush Hour app.

**Comment by eson57** [comment]
Can I install QtLocation only? I want to test translation before I upload.

**Comment by Rikujolla** [comment]
I hope, the problem is fixed in the version 0.0.4. Please install that version. I can't test that bug fix before I factory reset my Jolla 1.
Should be ok to pkcon install qt5-qtlocation also. I have not tested that.

**Comment by eson57** [comment]
Yes, 0.0.4 is working fine. Swedish translation looks good to.  ;)
The light blue icons is not very visible, maybe you should try a little bit darker or add some background?
